MechanicsCore/src/main/java/me/deecaad/core/mechanics/conditions/OutsideCondition.java

```diff
@@ -0,0 +1,30 @@
+package me.deecaad.core.mechanics.conditions;
+
+import me.deecaad.core.file.SerializeData;
+import me.deecaad.core.file.SerializerException;
+import me.deecaad.core.mechanics.CastData;
+import me.deecaad.core.mechanics.conditions.Condition;
+import org.bukkit.Location;
+import org.jetbrains.annotations.NotNull;
+
+public class OutsideCondition extends Condition {
```

Comment:
Have to reject this one... This is already done and more efficient by using the [LightLevelCondition](https://github.com/WeaponMechanics/MechanicsMain/wiki/LightLevelCondition) and making sure it is 16 (maximum light)